modules/ipi-install-configuring-nodes.adoc

```diff
@@ -87,3 +87,33 @@ To enable Secure Boot manually, refer to the hardware guide for the node and exe
 ====
 Red Hat does not support Secure Boot with self-generated keys.
 ====
+
+[discrete]
+== Configuring the Compatibility Support Module for Fujitsu iRMC
+
+The Compatibility Support Module (CSM) configuration provides the support for legacy BIOS backward compatibility with UEFI systems.
+
+[NOTE]
+====
+When you are deploying with iRMC, you must configure the CSM, otherwise the cluster installation might fail.
+====
+
+[IMPORTANT]
+====
+Before you configure the CSM, ensure that you have disabled the `Secure Boot Control`, which is available under `Security` -> `Secure Boot Configuration`.
```

Comment:
You could split this into two sentences, to facilitate localisation. For example:

>  Before you configure the CSM, ensure that you have disabled Secure Boot Control. You can disable Secure Boot control under **Security** -> **Secure Boot Configuration**.

In the above example, I have also updated the back-tick references to bold font instead, because these I assume are references to UI elements. Back-ticks are typically used to refer to code references. For the initial "Secure Boot Control" references in the example line I have removed the back ticks and not replaced them with bold font, because they don't refer to the UI element specifically, but instead to the feature name.

Reply:
> In the above example, I have also updated the back-tick references to bold font instead, because these I assume are references to UI elements. Back-ticks are typically used to refer to code references.

+1

> For the initial "Secure Boot Control" references in the example line I have removed the back ticks and not replaced them with bold font, because they don't refer to the UI element specifically, but instead to the feature name.

In fact, "Secure Boot Control" is both the UI element and the feature name, do we need to bold it?

Reply:
I updated "Secure Boot Control"  to bold. If this is not appropriate, please let me know.

Reply:
> In fact, "Secure Boot Control" is both the UI element and the feature name, do we need to bold it?

@hs0210 you could reference "Secure Boot Control" without using bold font when you are referring to the feature name and use bold font explicitly when you are referring to the UI element.

Reply:
+1